unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -165,6 +165,20 @@ public UTF8String toLowerCase() {
     return UTF8String.fromString(toString().toLowerCase());
   }
 
+  public UTF8String initCap(final byte[] byteArr) {
+    if (byteArr == null) return null;
+    if (byteArr[0] <= 122 && byteArr[0] >= 97) //a-z
```

Comment:
Europe language is not supported in this method. It is not supported in Java StringBuilder toUpper function. 
Since few words start with the special characters, and this kind of characters canbe converted in to English letters, for example ö => oe , so I think your suggestions will not be accepted.